pkg/cloud/services/networking/trunk_test.go

```diff
@@ -24,7 +24,7 @@ import (
 	. "github.com/onsi/gomega"
 
 	infrav1 "sigs.k8s.io/cluster-api-provider-openstack/api/v1alpha6"
-	"sigs.k8s.io/cluster-api-provider-openstack/pkg/cloud/services/networking/mock_networking"
+	mock_clients "sigs.k8s.io/cluster-api-provider-openstack/pkg/clients/mock"
```

Comment:
Probably it is okay to not define any alias here - we could just use `mock.`
What do you think?
mock_clients is also fine.

Reply:
You're right, and this would be cleaner. I'll want to do it consistently, though. Let me try an update across the whole PR to see if 'mock' conflicts anywhere and if not I'll update it.

Reply:
Looks like this request is still open @mdbooth

Reply:
Good point. Done.